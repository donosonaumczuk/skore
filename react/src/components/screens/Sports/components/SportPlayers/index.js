import React from 'react';
import Proptypes from 'prop-types';
import i18next from 'i18next';

const SportPlayers = ({ playersQuantity }) => {
    return (
            <div className="offset-1 col my-auto">
                <i className="name-label fas fa-users mr-2"></i>
                <label>{i18next.t('sports.playersPerTeam')}</label>
                 <span>{` ${playersQuantity}`}</span>
            </div>
    );
}

SportPlayers.propTypes = {
    playersQuantity: Proptypes.number.isRequired
}

export default SportPlayers;