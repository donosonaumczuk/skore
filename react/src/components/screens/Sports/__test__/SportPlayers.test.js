import React from 'react';
import renderer from 'react-test-renderer';
import SportPlayers from '../components/SportPlayers';

test('SportPlayers snapshot test', () => {
    //set up
    const playersQuantity = 2;
    
    //execution
    const component = renderer.create(
      <SportPlayers playersQuantity={playersQuantity} />,
    );
    let tree = component.toJSON();

    //postconditions
    expect(tree).toMatchSnapshot();
});