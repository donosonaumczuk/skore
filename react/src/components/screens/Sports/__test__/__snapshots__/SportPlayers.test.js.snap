// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SportPlayers snapshot test 1`] = `
<div
  className="offset-1 col my-auto"
>
  <i
    className="name-label fas fa-users mr-2"
  />
  <label />
  <span>
     2
  </span>
</div>
`;
